docs/standalone-signaling-api-v1.md

```diff
@@ -1416,6 +1416,151 @@ The signaling server provides an internal API that can be called from Nextcloud
+
+Below is an example response of the serverinfo endpoint with a connected Janus
+server:
+
+    {
+      "version": "1.2.3",
+      "features": [
+        "feature-1",
+        "feature-2",
+        ...
+        "serverinfo",
+        ...
+      ],
+      "sfu": {
+        "mode": "janus",
+        "janus": {
+          "url": "ws://localhost:8188/",
+          "connected": true,
+          "name": "Janus WebRTC Server",
+          "version": "1.3.1",
+          "author": "Meetecho s.r.l.",
+          "datachannels": true,
+          "fulltrickle": true,
+          "localip": "192.168.0.1",
+          "ipv6": false,
+          "videoroom": {
+            "name": "JANUS VideoRoom plugin",
+            "version": "0.0.10",
+            "author": "Meetecho s.r.l."
+          }
+        }
+      }
+    }
+
+If the backend is not connected, the value of `connected` in `janus` will be
+`false` and most other entries will be missing.
+
+
+### Example response with signaling proxy backends
+
+Below is an example response of the serverinfo endpoint with multiple signaling
+proxies:
+
+    {
+      "version": "1.2.3",
+      "features": [
+        "feature-1",
+        "feature-2",
+        ...
+        "serverinfo",
+        ...
+      ],
+      "sfu": {
+        "mode": "proxy",
+        "proxies": [
+          {
+            "url": "https://proxy.domain.tld/",
+            "ip": "192.168.0.1",
+            "connected": true,
+            "temporary": false,
+            "shutdown": false,
+            "uptime": "2025-03-05T18:09:35.435902408+01:00",
+            "version": "2.3.4",
+            "features": [
+              "proxy-feature-1",
+              "proxy-feature-2",
+              ...
+            ],
+            "country": "DE",
+            "load": 0,
+            "bandwidth": {
+              "incoming": 0,
+              "outgoing": 0
+            }
+          },
+          {
+            "url": "https://proxy.domain.tld/",
+            "ip": "192.168.0.2",
+            "connected": false,
+            "temporary": false
+          }
+        ]
+      }
+    }
+
+The `ip` field will only be present if DNS discovery is enabled for resolving
+proxy urls.
+`uptime` is the ISO8601 time since the connection was established to the proxy.
+`country` will only be returned if configured on the proxy.
+`load` is an arbitrary value used to order signaling proxies when selecting the
+proxy to use for publishing new streams.
+`bandwidth` contains the percentage of incoming / outgoing bandwith utilization
+for streams on the proxy. Only present if a bandwidth limit is configured on
+the proxy.
+
+
+### Dialout session
+
+If a SIP bridge with support for dial-out is connected, the serverinfo response
+will contain an additional property `dialout` with the following contents:
+
+    [
+      {
+        "sessionid": "the-session-id",
+        "connected": true,
+        "address": "192.168.1.0",
+        "useragent": "spreed-webrtc-sip-bridge/1.2.3",
```

Comment:
string split on / for version?
Or can we add a dedicated field?

Reply:
There is no version information that could be extracted from the `hello` request, so I will extract the version from the useragent and provide it as separate field.

Reply:
Thanks